docs/operator/use-cases/node-labels-and-annotations.md

```diff
@@ -0,0 +1,40 @@
+# Denying specific user-defined labels or annotations on Nodes
+
+**Important note**
```

Comment:
Maybe we could use the blockquotes for this important now, and maybe at the end of the section?

Reply:
done :)